src/BenchmarkDotNet.Core/Environments/HostEnvironmentInfo.cs

```diff
@@ -11,53 +11,55 @@ namespace BenchmarkDotNet.Environments
     // this class is used by our auto-generated benchmark program, 
     // keep it in mind if you want to do some renaming
     // you can finde the source code at Templates\BenchmarkProgram.txt
-    public sealed class HostEnvironmentInfo : BenchmarkEnvironmentInfo
+    public class HostEnvironmentInfo : BenchmarkEnvironmentInfo
     {
         public static readonly CultureInfo MainCultureInfo;
 
         private static HostEnvironmentInfo Current;
 
-        public string BenchmarkDotNetCaption { get; }
+        public string BenchmarkDotNetCaption { get; protected set; }
 
-        public string BenchmarkDotNetVersion { get; }
+        public string BenchmarkDotNetVersion { get; protected set; }
 
-        public string OsVersion { get; }
+        public string OsVersion { get; protected set; }
 
         /// <summary>
         /// is expensive to call (1s)
         /// </summary>
-        public Lazy<string> ProcessorName { get; }
+        public Lazy<string> ProcessorName { get; protected set; }
 
-        public int ProcessorCount { get; }
+        public int ProcessorCount { get; protected set; }
 
-        public string JitModules { get; }
+        public string JitModules { get; protected set; }
 
         /// <summary>
         /// is expensive to call (creates new process)
         /// </summary>
-        public Lazy<string> DotNetCliVersion { get; }
+        public Lazy<string> DotNetCliVersion { get; protected set; }
 
         /// <summary>
         /// The frequency of the timer as the number of ticks per second.
         /// </summary>
-        public Frequency ChronometerFrequency { get; }
+        public Frequency ChronometerFrequency { get; protected set; }
+        public TimeInterval ChronometerResolution { get; protected set; }
```

Comment:
It's better to have a method for `Resolution` instead of a property because we want to have the following invariant: `Resolution * Frequency = 1 second`. The implementation is trivial, just use `ChronometerFrequency.ToResolution()`.